tests/h/search/index_test.py

```diff
@@ -94,6 +102,111 @@ def test_you_can_filter_annotations_by_creation_time(self, factories, index, sea
+
+        index(annotation)
+
+        response = search.filter("term", user_raw="acct:someone@example.com").execute()
+
+        assert SearchResponseWithIDs([annotation.id]) == response
+
+    def test_you_can_filter_annotations_by_tags(self, factories, index, search):
+        annotation = factories.Annotation.build(id="test_annotation_id", tags=["tag1", "tag2"])
+
+        index(annotation)
+
+        response1 = search.filter("term", tags=["tag1"]).execute()
+        response2 = search.filter("term", tags=["tag2"]).execute()
+
+        assert SearchResponseWithIDs([annotation.id]) == response1
+        assert SearchResponseWithIDs([annotation.id]) == response2
+
+    def test_you_can_filter_annotations_by_tags_raw(self, factories, index, search):
+        annotation = factories.Annotation.build(id="test_annotation_id", tags=["tag1", "tag2"])
+
+        index(annotation)
+
+        response1 = search.filter("term", tags_raw=["tag1"]).execute()
+        response2 = search.filter("term", tags_raw=["tag2"]).execute()
+
+        assert SearchResponseWithIDs([annotation.id]) == response1
+        assert SearchResponseWithIDs([annotation.id]) == response2
+
+    def test_you_can_filter_annotations_by_uri(self, es_client, factories, index, search):
+        my_uri = 'http://example.com/anything/i/like?ex=something'
+        annotation = factories.Annotation.build(id="test_annotation_id", target_uri=my_uri)
+
+        index(annotation)
+
+        response = search.filter("term", uri='example.com/anything/i/like').execute()
+
+        assert SearchResponseWithIDs([annotation.id]) == response
+
+    def test_you_can_filter_annotations_by_text(self, factories, index, search):
+        annotation = factories.Annotation.build(id="test_annotation_id", text="text")
+
+        index(annotation)
+
+        response = search.filter("term", text="text").execute()
+
+        assert SearchResponseWithIDs([annotation.id]) == response
+
+    def test_you_can_filter_annotations_by_group(self, factories, index, search):
+        annotation = factories.Annotation.build(id="test_annotation_id", groupid="some_group")
+
+        index(annotation)
+
+        response = search.filter("term", group="some_group").execute()
+
+        assert SearchResponseWithIDs([annotation.id]) == response
+
+    def test_you_can_filter_annotations_by_shared(self, factories, index, search):
+        annotation = factories.Annotation.build(id="test_annotation_id", shared=False)
+
+        index(annotation)
+
+        response = search.filter("term", shared=False).execute()
+
+        assert SearchResponseWithIDs([annotation.id]) == response
+
+    def test_you_can_filter_annotations_by_thread_ids(self, es_client, factories, index, search):
+        annotation1 = factories.Annotation.build(id="test_annotation_id1")
+        annotation2 = factories.Annotation.build(id="test_annotation_id2", references=[annotation1.id], thread=[annotation1])
```

Comment:
Why is it necessary to pass both `references` and `thread` here?

Reply:
We don't. Thanks for catching that.